<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogBRISK</class>
 <widget class="QDialog" name="DialogBRISK">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1" colspan="2">
    <widget class="QCheckBox" name="cBORBKeyPoints">
     <property name="text">
      <string>orb keypoints</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>thresh</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QSlider" name="sliderThresh">
     <property name="maximum">
      <number>128</number>
     </property>
     <property name="value">
      <number>30</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>octaves</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QSlider" name="sliderOctaves">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>8</number>
     </property>
     <property name="sliderPosition">
      <number>3</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="QCheckBox" name="cBDrawKeyPoints">
     <property name="text">
      <string>draw keypoints</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>patternScale</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QDoubleSpinBox" name="sBPatternScale">
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>184</width>
       <height>182</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QCheckBox" name="cBFASTKeyPoints">
     <property name="text">
      <string>fast keypoints</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="8">
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
